### PR TITLE
Disable EncryptDecrypt_Read() test on the Home editions of Windows

### PR DESCRIPTION
**Pull request by juliushardt**
`build-tests.cmd` currently fails on the Home editions of Windows, e.g. Windows 10 Home, because `System.IO.Tests.EncryptDecrypt.EncryptDecrypt_Read()` calls `File.Encrypt(string path)` which is not supported on these editions and thus throws an `IOException`. This PR disables the test for `File.Encrypt` on Windows 7/8.1/10 Home, so that all tests succeed.

### CODE REVIEW

File: src/System.IO.FileSystem/tests/File/EncryptDecrypt.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public static void EncryptDecrypt_NotSupported()
             Assert.Throws<PlatformNotSupportedException>(() => File.Decrypt(null));
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer), nameof(PlatformDetection.IsNotWindowsHomeEdition))]
```
Comment (ViktorHofer) [inline src/System.IO.FileSystem/tests/File/EncryptDecrypt.cs]: Is that the right Syntax to check for two conditions?

Comment (ViktorHofer) [reply src/System.IO.FileSystem/tests/File/EncryptDecrypt.cs]: Please add a comment explaining why we skip the test on home editions.

Comment (danmosemsft) [reply src/System.IO.FileSystem/tests/File/EncryptDecrypt.cs]: Yes it is the correct syntax

**Comment by ViktorHofer** [comment]
Thanks @juliushardt!

**Pull request merged**